src/views/record-browser/record-browser.scss

```diff
@@ -62,4 +62,57 @@ $sidebar-width: 300px;
 			&.sep { color: $gray-medium; }
 		}
 	}
+
+	thead {
+		/* remove formatting for delete button column header cell */
+		th:first-child {
+			color: transparent;
+			background: transparent;
+			border-bottom: 0px;
```

Comment:
Units are unnecessary when the value is zero. `0`, `0em`, `0px`, `0%`, etc are all equivalent